db/migrations/02_create_peeps_table.sql

```diff
@@ -0,0 +1,3 @@
+CREATE TABLE peeps (id SERIAL PRIMARY KEY, text VARCHAR(280))
+ALTER TABLE "peeps" ADD COLUMN "user_id" varchar(60);
+ALTER TABLE "peeps" ADD COLUMN "time_stamp" varchar(60);
```

Comment:
Maybe think about data types that the db can provide for you and make sometimes your life simple. In this case timestamp data type in pg.